website/static/js/paginator.js

```diff
@@ -7,6 +7,10 @@ var oop = require('js/oop');
 var MAX_PAGES_ON_PAGINATOR = 7;
 var MAX_PAGES_ON_PAGINATOR_SIDE = 5;
 
+
+
+/* jshint ignore:start */
```

Comment:
Oh, I see; there are multiple places that a function is created within a loop. Ok, this is fine for now; disregard previous comment.